save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/LnkUserOrganization.kt

```diff
@@ -0,0 +1,21 @@
+package org.cqfn.save.entities
+
+import javax.persistence.Entity
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+
+/**
+ * @property organization
+ * @property user in organization
+ */
+@Entity
+class LnkUserOrganization(
```

Comment:
Do we need something like `isAdmin` or a more general `organizationRole` here?

Reply:
I thought about it. But I decided that at this stage we don't need it. If needed in the future, we will add.